tvmaze/episode.go

```diff
@@ -51,7 +52,10 @@ func (c Client) GetEpisode(s Show, season int, episode int) (*Episode, error) {
 	})
 
 	var epOut Episode
-	if err := c.get(url, &epOut); err != nil {
+	if status, err := c.get(url, &epOut); err != nil {
```

Comment:
I tried masking the 404 error for other cases that might've been returning a nil error but this is a little awkward -- and maybe an error is appropriate?

Reply:
this was awkward so I backtracked